crates/napi/src/threadsafe_function.rs

```diff
@@ -307,20 +337,19 @@ impl<T: 'static, ES: ErrorStrategy::T> ThreadsafeFunction<T, ES> {
         async_resource_name,
         max_queue_size,
         1,
-        ptr::null_mut(),
+        Arc::into_raw(Pin::into_inner(handle.clone()))
+          .cast_mut()
```

Comment:
This API was added in `1.65.0`: https://doc.rust-lang.org/std/primitive.pointer.html#method.cast_mut
It breaks the MSRV